frontend/src/scenes/insights/insightVizDataLogic.ts

```diff
@@ -162,6 +191,26 @@ export const insightVizDataLogic = kea<insightVizDataLogicType>([
                 }
             }
         },
+        loadData: async ({ queryId }, breakpoint) => {
+            actions.setTimedOutQueryId(null)
+
+            await breakpoint(SHOW_TIMEOUT_MESSAGE_AFTER)
+
+            if (!!values.insightDataLoading) {
+                actions.setTimedOutQueryId(queryId)
+                const tags = {
+                    kind: values.querySource?.kind,
+                    scene: sceneLogic.isMounted() ? sceneLogic.values.scene : null,
+                }
+                posthog.capture('insight timeout message shown', tags)
```

Comment:
@yakkomajuri Are you using this event for performance analysis somehow (or know who'd use this event)? Asking, as I'm changing the tag from `insight: TRENDS/FUNNELS/...` to `kind: TrendsQuery/FunnelsQuery/...` and wasn't sure if we're using the existing tag somewhere.